examples/AzureSDKDemoSwift/Source/BlobDownloadViewController.swift

```diff
@@ -180,8 +180,8 @@ extension BlobDownloadViewController: UITableViewDelegate, UITableViewDataSource
         guard let blobName = cell.keyLabel.text else { return }
         guard let containerName = AppConstants.videoContainer else { return }
         guard let blobClient = blobClient else { return }
-        let destination = StorageBlobClient.PathHelper
-            .localUrl(inDirectory: StorageBlobClient.PathHelper.cacheDir, forBlob: blobName, inContainer: containerName)
+        // FIXME: Major bug--cannot rehydrate transfers due to hard coding of local paths
```

Comment:
Uploads and downloads cannot be rehydrated due to changes made to use hardcoded URLs. You can pause and resume but if you try and close the app and reload, both operations will fail. It will need to be fixed in a separate PR.

Reply:
Definitely will want to chat and understand why this is the case. I'm okay addressing in a followup PR.